nixos/modules/config/nix.nix

```diff
@@ -333,7 +333,15 @@ in
               description = ''
                 The set of features supported by the machine. Derivations
                 can express dependencies on system features through the
-                `requiredSystemFeatures` attribute.
+                `requiredSystemFeatures` attribute. Also see {option}`nix.settings.extra-system-features`.
+              '';
+            };
+
+            extra-system-features = mkOption {
```

Comment:
Given that `settings.system-features` already will do merge across the whole of the nixos module system, why do we need to document this option?